docs/api/image-view.md

```diff
@@ -0,0 +1,53 @@
+# ImageView
+
+> A View that displays an image.
+
+Process: [Main](../glossary.md#main-process)
+
+This module cannot be used until the `ready` event of the `app`
+module is emitted.
```

Comment:
```suggestion
module is emitted.

Useful for showing splash screens that will be swapped for `WebContentsView`s
when the content finishes loading.
```

I think it would be useful to document why someone would use an `ImageView`.